compose.dev.yaml

```
@@ -0,0 +1,9 @@
+services:
+  db:
+    image: postgres:16
+    environment:
+      POSTGRES_DB: meme
+      POSTGRES_USER: atka
+      POSTGRES_PASSWORD: atka
+    ports:
+      - "5432:5432"
```

Good practice is to always bind to localhost:

```suggestion
      - "localhost:5432:5432"
```

This is especially useful for people with a public IP address (e.g. some dormitories) because anyone could try to bruteforce these. My DB got hacked and backdoored in half an hour when I've set up a prebuilt container with default credentials.

---

lgtm, i honestly assumed dev machines have firewalls, but it's a great idea :D